Author: <jame@rocasa.us>
Description: Remove the xtrn/sbl/sbl & xtrn/sbl2smb, xtrn/sbl/sblpack, & xtrn/sbl/smb2sbl
files that come with the source archive, as they are actually only scripts
that attempt to locate the real executables in a build directory, which is
not needed for the debian package.
--- synchronet-3.14a.orig/xtrn/sbl/sbl
+++ /dev/null
@@ -1,15 +0,0 @@
-#!/bin/sh
-os=`uname | tr "[A-Z]" "[a-z]"`
-exename=`basename $0`
-dirname=`dirname $0`
-if [ -x $dirname\/gcc.$os.exe.release/$exename ]
-	then exec $dirname\/gcc.$os.exe.release/$exename $@
-elif [ -x $dirname\/gcc.$os.exe.debug/$exename ]
-	then exec $dirname\/gcc.$os.exe.debug/$exename $@
-elif [ -x $dirname\/*.$os.exe.release/$exename ]
-	then exec $dirname\/*.$os.exe.release/$exename $@
-elif [ -x $dirname\/*.$os.exe.debug/$exename ]
-	then exec $dirname\/*.$os.exe.debug/$exename $@
-elif [ -x $dirname\/*.$os.exe.*/$exename ]
-	then exec $dirname\/*.$os.exe.*/$exename $@
-fi
--- synchronet-3.14a.orig/xtrn/sbl/sbl2smb
+++ /dev/null
@@ -1,15 +0,0 @@
-#!/bin/sh
-os=`uname | tr "[A-Z]" "[a-z]"`
-exename=`basename $0`
-dirname=`dirname $0`
-if [ -x $dirname\/gcc.$os.exe.release/$exename ]
-	then exec $dirname\/gcc.$os.exe.release/$exename $@
-elif [ -x $dirname\/gcc.$os.exe.debug/$exename ]
-	then exec $dirname\/gcc.$os.exe.debug/$exename $@
-elif [ -x $dirname\/*.$os.exe.release/$exename ]
-	then exec $dirname\/*.$os.exe.release/$exename $@
-elif [ -x $dirname\/*.$os.exe.debug/$exename ]
-	then exec $dirname\/*.$os.exe.debug/$exename $@
-elif [ -x $dirname\/*.$os.exe.*/$exename ]
-	then exec $dirname\/*.$os.exe.*/$exename $@
-fi
--- synchronet-3.14a.orig/xtrn/sbl/sblpack
+++ /dev/null
@@ -1,15 +0,0 @@
-#!/bin/sh
-os=`uname | tr "[A-Z]" "[a-z]"`
-exename=`basename $0`
-dirname=`dirname $0`
-if [ -x $dirname\/gcc.$os.exe.release/$exename ]
-	then exec $dirname\/gcc.$os.exe.release/$exename $@
-elif [ -x $dirname\/gcc.$os.exe.debug/$exename ]
-	then exec $dirname\/gcc.$os.exe.debug/$exename $@
-elif [ -x $dirname\/*.$os.exe.release/$exename ]
-	then exec $dirname\/*.$os.exe.release/$exename $@
-elif [ -x $dirname\/*.$os.exe.debug/$exename ]
-	then exec $dirname\/*.$os.exe.debug/$exename $@
-elif [ -x $dirname\/*.$os.exe.*/$exename ]
-	then exec $dirname\/*.$os.exe.*/$exename $@
-fi
--- synchronet-3.14a.orig/xtrn/sbl/smb2sbl
+++ /dev/null
@@ -1,15 +0,0 @@
-#!/bin/sh
-os=`uname | tr "[A-Z]" "[a-z]"`
-exename=`basename $0`
-dirname=`dirname $0`
-if [ -x $dirname\/gcc.$os.exe.release/$exename ]
-	then exec $dirname\/gcc.$os.exe.release/$exename $@
-elif [ -x $dirname\/gcc.$os.exe.debug/$exename ]
-	then exec $dirname\/gcc.$os.exe.debug/$exename $@
-elif [ -x $dirname\/*.$os.exe.release/$exename ]
-	then exec $dirname\/*.$os.exe.release/$exename $@
-elif [ -x $dirname\/*.$os.exe.debug/$exename ]
-	then exec $dirname\/*.$os.exe.debug/$exename $@
-elif [ -x $dirname\/*.$os.exe.*/$exename ]
-	then exec $dirname\/*.$os.exe.*/$exename $@
-fi
